train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java

```diff
@@ -25,4 +29,15 @@ public TrainUser getUser() {
 		return user;
 	}
 
+	public void startOperation() {
+		Random r = new Random();
```

Comment:
Random returns a value between 0 and 1

Reply:
true dat, ill fix it

Reply:
ill reviewd it, your ideas make no sense